app.json

```diff
@@ -10,7 +10,7 @@
     "MATTERMOST_DOWNLOAD_URI": {
         "description": "The URI to the compiled Mattermost binary you would like to run. Defaults to latest release. See https://www.mattermost.org/download/ for download links.",
         "required": true,
-        "value": "https://releases.mattermost.com/5.26.0/mattermost-5.26.0-linux-amd64.tar.gz"
+        "value": "https://releases.mattermost.com/5.32.1/mattermost-5.32.1-linux-amd64.tar.gz"
```

Comment:
we have released 5.35.3, maybe you can update to that

Reply:
@cpanato I wonder if we'd merge this, and the version can be updated with another PR?

Reply:
yes we can....